data-raw/healthy-lives/behavioural-risk-factors/alcohol-misuse.R

```diff
@@ -0,0 +1,87 @@
+# Load packages
+library(tidyverse)
+library(readxl)
+library(sf)
+library(httr)
+library(httr)
+library(dplyr)
+
+# URL of the Excel file
+excel_url <- "https://www.ons.gov.uk/file?uri=/peoplepopulationandcommunity/birthsdeathsandmarriages/deaths/datasets/alcoholspecificdeathsinenglandandwalesbylocalauthority/current/alcoholspecificdeathsbylocalauthority2022.xlsx"
```

Comment:
Please rename your variables here and throughout the script to ensure good naming convention. See Guide above for what makes a good object name. In general they need to be descriptive of the data/object yet concise.